components/ProgressIndicator/__stories__/ProgressIndicator.stories.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { withKnobs, text } from '@storybook/addon-knobs';
+
+import ProgressIndicator from '../ProgressIndicator';
+
+storiesOf('ProgressIndicator', module)
+  .addDecorator(withKnobs)
+  .add(
+    'default',
+    withInfo()(() => <ProgressIndicator>{text('children', 'string or .node')}</ProgressIndicator>),
```

Comment:
looks like you've gotta match the story with the existing API

Reply:
What was the issue? You won't be able to see the component properly there until you match the component's expected usage.